jsapp/js/account/stripe.api.ts

```diff
@@ -0,0 +1,82 @@
+import {fetchGet, fetchPost} from 'jsapp/js/api';
+import type {PaginatedResponse} from 'js/dataInterface';
+
+export interface BaseProduct {
+  id: string;
+  name: string;
+  description: string;
+  type: string;
+  metadata: {[key: string]: string};
+}
+
+export interface BasePrice {
+  id: string;
+  nickname: string;
+  currency: string;
+  type: string;
+  unit_amount: number;
+  human_readable_price: string;
+  metadata: {[key: string]: string};
+}
+
+export interface BaseSubscription {
+  id: number;
+  price: Product;
+}
+
+export interface Organization {
+  uid: string;
+  name: string;
+  is_active: boolean;
+  created: string;
+  modified: string;
+  slug: string;
+}
+
+export interface Product extends BaseProduct {
+  prices: Array<BasePrice>;
+}
+
+export interface Price extends BaseProduct {
+  prices: BasePrice;
+}
+
+export interface Checkout {
+  url: string;
+}
+
+export interface Portal {
+  url: string;
+}
+
+const PRODUCTS_URL = '/api/v2/stripe/products/';
+const SUBSCRIPTION_URL = '/api/v2/stripe/subscriptions/';
+const ORGANIZATION_URL = '/api/v2/organizations/';
+const CHECKOUT_URL = '/api/v2/stripe/checkout-link';
+const PORTAL_URL = '/api/v2/stripe/customer-portal';
```

Comment:
I was thinking about this for some time last week. Do you think it would make sense to have a single file somewhere at the top directory with a list of all API endpoints we use? It would be nice to use these instead of hardcoding them

Reply:
Yeah I think that would be really helpful because then it reduces having to update a URL in multiple locations/files if the URL changes. I do think it would make sense to separate endpoints based on use, for example, this would live in a Stripe Endpoint file and we have a directory for all endpoint files

Reply:
Separate files are nice, but on the other hand the number of API endpoints we have is not so big and it might fit nicely in single file :)

Reply:
Perhaps next to `api.ts` as `api.endpoints.ts`?

Reply:
Thats fair, I wasn't sure how many we were talking about in total.  I do like `api.endpoints.ts`